android/app/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,2 @@
+connection.project.dir=..
+eclipse.preferences.version=1
```

Comment:
Should this have been removed along with the other Eclipse files?

Reply:
removed